packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/ManifestFactory.kt

```diff
@@ -11,14 +10,9 @@ import org.json.JSONObject
  * Utility methods for parsing a JSON manifest using the correct [UpdateManifest] implementation.
  */
 object ManifestFactory {
-  private val TAG = ManifestFactory::class.java.simpleName
-
   @Throws(Exception::class)
   fun getManifest(manifestJson: JSONObject, responseHeaderData: ResponseHeaderData, extensions: JSONObject?, configuration: UpdatesConfiguration?): UpdateManifest {
     return when (val expoProtocolVersion = responseHeaderData.protocolVersion) {
-      null -> {
-        LegacyUpdateManifest.fromLegacyManifest(LegacyManifest(manifestJson), configuration!!)
-      }
       0, 1 -> {
```

Comment:
This means that in the past, 0 or 1 was always specified. Going forward, we can throw a nice error when null, especially if the manifest "looks like" a legacy manifest